packages/testkit/package.json

```
@@ -17,13 +27,14 @@
   },
   "scripts": {
     "test:e2e": "npx playwright test",
-    "build": "tsc",
+    "build": "tsc && tsc --project tsconfig.esm.json",
     "start-server": "yarn lerna run storybook --scope=monday-ui-react-core"
   },
   "bugs": {
     "url": "https://github.com/mondaycom/vibe/issues"
   },
   "devDependencies": {
-    "@playwright/test": "1.45.3"
+    "@playwright/test": "1.45.3",
```

is `playwright`'s version locked by accident or is that intended? 
```suggestion
    "@playwright/test": "^1.45.3",
```

---

@YossiSaadi yes it's locked on purpose since it is the version used currently in some of our repo's. we'd need to have a separate task for bumping versions, synchronously to avoid mismatch.

---

got ya, cool 💯